saleor/core/notification/utils.py

```diff
@@ -1,10 +1,16 @@
 from django.contrib.sites.models import Site
+from django.contrib.staticfiles.storage import staticfiles_storage
+
+from ..utils import build_absolute_uri
 
 
 def get_site_context():
     site: Site = Site.objects.get_current()
     site_context = {
         "domain": site.domain,
         "site_name": site.name,
+        "logo_url": build_absolute_uri(
+            staticfiles_storage.url("images/saleor-logo.png")
```

Comment:
Let's change the filename to `saleor-logo-sign.png` (or something similar). This will prevent the situation of unintended replacing it with a full logo in the future.